mathisfun/Complex.cpp

```diff
@@ -0,0 +1,50 @@
+#include "Complex.h"
+
+#include <iostream>
+
+Complex::Complex()
+	:m_real(0), m_imaginary(0)
+{}
```

Comment:
To let people know you've intentionally left this function blank, use a comment.
Otherwise, people will assume you left it blank by accident.